tests/test_sky_model.py

```diff
@@ -0,0 +1,34 @@
+import numpy as np
+import galsim
+from imsim import SkyModel
+
+
+def test_sky_model():
```

Comment:
This test looks fine.  But I'll note that we already had a file test_skyModel.py.  Is anything in there reusable, modifying to the new syntax?  If not we should either remove it or mark which ones are still aspirational.

Reply:
I was thinking of just removing test_skyModel.py since almost all of the tests are specific to how that old skyModel.py code was written, i.e., using that `NoiseAndBackgroundBase` class from sims_GalSimInterface.  So, I'll just go ahead and remove it.